cmd/tools/grafana/grafana.go

```diff
@@ -1558,7 +1631,7 @@ func addImportCustomizeFlags(commands ...*cobra.Command) {
 		cmd.PersistentFlags().BoolVar(&opts.forceImport, "force", false,
 			"Import even if the datasource name is not defined in Grafana")
 		cmd.PersistentFlags().StringVar(&opts.customCluster, "cluster-label", "",
-			"Rewrite all panel expressions to use the specified cluster label instead of the default 'cluster'")
+			"Rewrite all panel expressions to add the specified cluster label and specified cluster variable along with the default 'cluster'")
```

Comment:
Since some folks are already using the `cluster-label` CLI argument to replace the default cluster label. We should create a new CLI argument to add a new variable. Maybe something like `--add-cluster-label` or `--new-cluster-label` That way customers can continue with their previous workflow and opt into this new one with a new CLI arg